packages/contractkit/test/utils.ts

```diff
@@ -40,24 +40,31 @@ export function getGasFeeRecipient(networkName: string) {
   switch (networkName) {
     case 'alfajoresstaging':
       return '0xFe6B1360Fb2D31B4574d05c1d8Af29053fE28F1A'
+    case 'alfajores':
+      // Seems to work
+      return '0xFe6B1360Fb2D31B4574d05c1d8Af29053fE28F1A'
```

Comment:
Just curious: why does it work for this to be the same as alfajoresstaging?

Reply:
> Just curious: why does it work for this to be the same as alfajoresstaging?

May be the etherbase generation approach is the same :)